srcpkgs/faust/patches/faust-musl-stacktrace.patch

```diff
@@ -0,0 +1,15 @@
+--- compiler/errors/errormsg.cpp-orig	2019-12-11 14:16:08.491068101 +0100
++++ compiler/errors/errormsg.cpp	2019-12-11 14:16:38.650069228 +0100
+@@ -43,9 +43,9 @@
+             gGlobal->printCompilationOptions(str);
+         }
+         str << ")\n";
+-#ifndef EMCC
+-        stacktrace(str, 20);
+-#endif
++// #ifndef EMCC
++//         stacktrace(str, 20);
++// #endif
```

Comment:
patching it like this disables stacktraces for glibc as well, you probably want to test for ``__GLIBC__``